microsoft-edge/webview2/index.md

```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
+> [!NOTE]
+> The WebView2 Preview is intended for early prototyping and to gather feedback from the developer community to helps us shape the future stable API. There will be [breaking changes](releasenotes.md), so it should NOT be used for production apps.
 
-For constrained environments we will also support a fixed version distribution model. In this model your application will select and package a specific WebView2 version. Updates to the WebView version will be the responsibility of the application and will be independent of any managed Microsoft update mechanisms. You should choose this model if it is crucial to be able to have absolute control over the browser version and update times your application is taking advantage of.
+## Hybrid Application Approach
 
-## Microsoft Edge WebView2 Runtime
+Developers often have to chose between building a web application or a native application. This decision hinges on the trade-off between Reach and Power.
 
-The Microsoft Edge WebView2 Runtime is a packaging of the browser binaries optimized for use by WebView2 applications. It will function stand alone or side-by-side with a client Edge Browser install. A single install of the run-time will support any number of WebView2 applications. Install of the runtime will not appear as a browser install to end-user, i.e. no desktop shortcuts, start menu entry, or protocol registration.
+![webnative](./media/WebView2/webnative.PNG)
 
-An application utilizing WebView2 must ensure the installation of the Microsoft Edge WebView2 Runtime has occurred. At application install time you should check the current install status, which can be determined by using [GetAvailableCoreWebView2BrowserVersionString](./reference/win32/0-9-488/webview2-idl.md#getavailablecorewebview2browserversionstring). If the WebView2 Runtime is not available, you should chain the Microsoft Edge WebView2 Runtime Installer to your install flow.
+Web applications allow for a broad reach. Web developers can reuse most, if not all of their code, across all different platforms. Native applications, however, can utilize the capabilities of the entire native platform.
 
-## Microsoft Edge WebView2 SDK
+Hybrid applications allow developers to enjoy the best of both worlds. Hybrid application developers benefit from the ubiquity and strength of the web platform as well as the power and full capabilities of the native platform.
 
-To utilize WebView2 within your app you'll need to install and reference the [WebView2 SDK](https://aka.ms/webviewnuget) in your application. Our NuGet releases will contain both a release and pre-release version. The release version contains the public APIs we currently intend to release to general availability.
+## WebView2 Benefits 
+![webviewreasons](./media/WebView2/webviewreasons.PNG)
```

Comment:
There are differences between the icon titles and the text below. For example, Fixed vs. Fixed distribution. Can you review please?

Reply:
Josh is going to finalize these two titles in his distribution doc - will fix when his change is complete